### ISSUE TITLE
User reports that OIDs printed with Internet Explorer and Chrome cannot be read by the TipToi pen

### ISSUE THREAD

**Issue by sidiandi** [sidiandi/ttaudio#24]
Test with other browsers than Firefox.

Drucken geht nur mit dem Firefox. Chrome und IE skalieren scheinbar die OID's. Durch Einstellungen konnte dies nicht kompensiert werden.

**Comment by cmorty** [comment]
I played around with Inkscape and I wasn't able to print any readable codes. I actually rewrote your printer test page to a SVG testpage. Opening the same SVG in FF and printing it worked just well. So I printed to pdfs and had a closer look. Inkscape just printed the dots,  whereas FF does some magic that could be some print optimization:
![grafik](https://cloud.githubusercontent.com/assets/2370091/21531174/3c6e151c-cd46-11e6-806b-ba6d145d1d25.png)
BTW: IE and Edge just rasterize the SVG. Therefore it's unlikely to work at all.
